editor/components/button/style.scss

```diff
@@ -1,8 +1,14 @@
 .editor-button {
 	background: none;
 	border: none;
+	outline: none;
 
 	&:disabled {
 		opacity: 0.6;
 	}
+
+	&.is-active {
```

Comment:
This means all the buttons will have the dark gray background when clicked (before releasing the mouse). I'm not sure we want this, do we?